apps/web/src/pages/Applications/ApplicationReviewPage/ApplicationReviewPage.tsx

```diff
@@ -49,22 +81,426 @@ export const getPageInfo: GetPageNavInfo = ({
+      <div data-h2-margin-bottom="base(x2)">
+        <p data-h2-margin-bottom="base(x1)">
+          {intl.formatMessage({
+            defaultMessage:
+              "Before continuing, it’s important that you understand the following notes about your application:",
+            id: "hX7Z1l",
+          })}
+        </p>
+        <ol>
+          <li>
+            <p data-h2-margin-bottom="base(x.5)">
+              {intl.formatMessage({
+                defaultMessage:
+                  "When you submit your application, a copy of your profile will be created as a part of your application.",
+                id: "WWoPd/",
+              })}
+            </p>
+          </li>
+          <li>
+            <p data-h2-margin-bottom="base(x.5)">
+              {intl.formatMessage({
+                defaultMessage:
+                  "Changes made to your profile after submitting will not be updated on your application.",
+                id: "RvafWw",
+              })}
+            </p>
+          </li>
+          <li>
+            <p data-h2-margin-bottom="base(x.5)">
+              {intl.formatMessage({
+                defaultMessage:
+                  "You are still encouraged to keep your profile up to date, as updated versions will be used at later steps of the hiring process.",
+                id: "TIU07I",
+              })}
+            </p>
+          </li>
+        </ol>
+      </div>
+      <ReviewSection
+        title={intl.formatMessage({
+          defaultMessage: "Your résumé",
+          id: "IYAKcM",
+        })}
+        path={editPaths.resume}
+      >
+        <TreeView.Root>
+          <TreeView.Head>
+            <Card title="" color="white" bold data-h2-margin-bottom="base(x1)">
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "Below is your résumé as it will be presented to hiring managers for these positions.",
+                  id: "cbx+9J",
+                })}
+              </p>
+            </Card>
+          </TreeView.Head>
+          {hasSomeExperience ? (
+            nonEmptyExperiences.map((experience) => (
+              <TreeView.Item key={experience.id}>
+                <div data-h2-margin="base(-x.5, 0)">
+                  <Accordion.Root type="single" collapsible>
+                    <ExperienceAccordion
+                      key={experience.id}
+                      experience={experience}
+                      headingLevel="h3"
+                      showSkills={false}
+                    />
+                  </Accordion.Root>
+                </div>
+              </TreeView.Item>
+            ))
+          ) : (
+            <Well>
+              <p data-h2-text-align="base(center)">
+                {intl.formatMessage({
+                  defaultMessage:
+                    "<strong>It looks like you haven't added any experiences to your résumé yet.</strong>",
```

Comment:
Within formatted messages `strong` just applies `data-h2-font-weight="base(700)"` so you could just add this to the `<p>` tag wrapping to simplify the string. Though, i'd argue this is not an appropriate use of bold text since[ it could be mistaken for a heading](https://style.ons.gov.uk/accessibility/text/#:~:text=Avoid%20using%20italics%20or%20bold%2C%20underlining%20text%20and%20overuse%20of%20capital%20letters%2C%20as%20the%20text%20could%20be%20mistaken%20for%20a%20heading%20and%20large%20amounts%20can%20be%20hard%20to%20read%20for%20users%20with%20visual%20impairments.%20Instead%2C%20create%20emphasis%20through%20what%20you%20write.) when it is overused.

Reply:
Fixed by [291476e](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6600/commits/291476ea31e08c3f5855dbb6dc1e0db4697471ec)